src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -79,7 +79,7 @@ const WorkspaceTabs = ({
 
   const tabs = [
     { name: 'dashboard', link: 'workspace-dashboard' },
-    ...(isGoogleWorkspace ? [{ name: 'data', link: 'workspace-data' }] : []),
+    ...([{ name: 'data', link: 'workspace-data' }]),
```

Comment:
Without the condition, the array and spread operator isn't necessary anymore.

```suggestion
    { name: 'data', link: 'workspace-data' },
```

Reply:
Bumped to use `isFeaturePreviewEnabled('workspace-data-service')`